tests/components/automatic/test_device_tracker.py

```diff
@@ -108,8 +104,7 @@ def get_trips(*args, **kwargs):
     session.get_trips.side_effect = get_trips
     session.refresh_token = "mock_refresh_token"
 
-    @asyncio.coroutine
-    def ws_connect():
+    async def ws_connect():
         return asyncio.Future()
```

Comment:
Is this still needed?  Now that the entire function is an awaitable, do you need to return this?

Reply:
Well, spotted sir! 👀 
Removed it.